frontend/src/Explorer/Util/String.js

```diff
@@ -9,3 +9,9 @@ exports.substituteImpl = function(str) {
     });
   };
 };
+
+exports.take = function (n) {
```

Comment:
@sectore found that `bower i purescript-strings#2.1.0 --save` works. I'll fix it after confirmation from @gromakovsky or @pva701.